### PR TITLE
Set "CSV_VERSION_PREV": "3.5.0-0.1679521107.p"

### PR DESCRIPTION
**Pull request by dmytro-ndp**
### What does this PR do?
It aligns Dev Spaces 3.6.0 to recent version of Dev Spaces 3.5.0-0.1679521107.p, and 3.7.0 to 3.6.0.

### What issues does this PR fix or reference?
https://issues.redhat.com/browse/CRW-4324

### CODE REVIEW

File: dependencies/job-config.json [hunk 1]
```diff
@@ -523,15 +523,15 @@
...
       },
       "3.7": {
         "CSV_VERSION": "3.7.0",
-        "CSV_VERSION_PREV": "3.5.0"
+        "CSV_VERSION_PREV": "3.6.0"
       },
       "3.x": {
         "CSV_VERSION": "3.7.0",
-        "CSV_VERSION_PREV": "3.5.0"
+        "CSV_VERSION_PREV": "3.6.0"
```
Comment (nickboldt) [inline dependencies/job-config.json:534]: There is no 3.6.0, so this should be the last released version

```suggestion
        "CSV_VERSION_PREV": "3.5.0-0.1679521107.p"
```

**Comment by nickboldt** [comment]
Unless you plan to do testing of 3.5 -> 3.6RC -> 3.7 CI upgrade path, the correct `_PREV` value for 3.7 and 3.x is the latest 3.5.0.fm.

Otherwise you'll get errors trying to install 3.7 as there is no released 3.6 yet to resolve the upgrade tree.

**Comment by dmytro-ndp** [comment]
If I were asked to test an upgrade to 3.7 CI, I would test the 3.5 -> 3.6RC -> 3.7 CI upgrade path as a more relevant option.

**Comment by nickboldt** [comment]
If we set 3.7 as a `replaces: 3.6` then you won't be able to install 3.7 without first installing 3.6.

we had this problem 6 weeks ago, and 12 weeks ago...

**Pull request merged**